Kickstarter-iOS/Views/Cells/PledgePaymentMethodsCell.swift

```diff
@@ -1,21 +1,25 @@
 import KsApi
 import Library
 import Prelude
+import Stripe
 import UIKit
 
 final class PledgePaymentMethodsCell: UITableViewCell, ValueCell {
+
   // MARK: - Properties
 
-  private let layout: UICollectionViewLayout = {
-    UICollectionViewFlowLayout()
-      |> \.scrollDirection .~ .horizontal
-  }()
+  private let applePayButton = PKPaymentButton()
```

Comment:
Yeah... I thought about it before choosing let. We should definitely choose some guidelines (`lazy var` vs `let`)  to avoid these inconsistencies, as I found other `let`s in the code... `lazy vars` give us the benefit of only allocating in memory when needed, saving space in memory. But in our case, most of the views are created immediately, so `lazy var`s lose their main benefit... For now I will change it to keep the code consistent but this is something we should keep in mind and talk about. Do you agree?